src/test/java/ua/softserveinc/tc/validator/EventValidatorTest.java

```diff
@@ -30,22 +30,16 @@
  */
 public class EventValidatorTest {
 
-    @Mock
     private EventDto eventDto;
 
-    @Mock
```

Comment:
why you delete Mock,?  I think it`s better to mock this object if you are not testing it,